### ISSUE TITLE
ReferenceError: Can't find variable: Platform

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#8670]
both old and new project `ReferenceError: Can't find variable: Platform` on iOS device 

```
  Dev mode.......... spa
    Pkg quasar........ v2.0.0-beta.10
    Pkg @quasar/app... v3.0.0-beta.9
    Transpiled JS..... yes (Babel)
```

<img width="980" alt="Screen Shot 2021-03-23 at 8 57 56 PM" src="https://user-images.githubusercontent.com/19895516/112210437-f907ac00-8c1a-11eb-8a6e-37a367b62c37.png">
<img width="811" alt="Screen Shot 2021-03-23 at 8 58 32 PM" src="https://user-images.githubusercontent.com/19895516/112210458-fe64f680-8c1a-11eb-914f-f83bb821ea10.png">

**Comment by rstoenescu** [comment]
https://github.com/quasarframework/quasar/issues/8469

**Comment by diadal** [comment]
this real bug guys check this ` Platform.is.winphone` is used but it was import as `client `


<img width="532" alt="Screen Shot 2021-03-25 at 9 02 04 AM" src="https://user-images.githubusercontent.com/19895516/112438947-ce1b7600-8d48-11eb-9a6d-7d4e477b3bb6.png">


```
import { client } from './Platform.js'
import { noop } from '../utils/event.js'
import getCssVar from '../utils/get-css-var.js'

let metaValue

function getProp () {
  return Platform.is.winphone
    ? 'msapplication-navbutton-color'
    : (
        Platform.is.safari
          ? 'apple-mobile-web-app-status-bar-style'
          : 'theme-color' // Chrome, Firefox OS, Opera, Vivaldi, ...
      )
}

function getMetaTag (v) {
  const els = document.getElementsByTagName('META')
  for (const i in els) {
    if (els[ i ].name === v) {
      return els[ i ]
    }
  }
}

function setColor (hexColor) {
  if (metaValue === void 0) {
    // cache it
    metaValue = getProp()
  }

  let metaTag = getMetaTag(metaValue)
  const newTag = metaTag === void 0

  if (newTag) {
    metaTag = document.createElement('meta')
    metaTag.setAttribute('name', metaValue)
  }

  metaTag.setAttribute('content', hexColor)

  if (newTag) {
    document.head.appendChild(metaTag)
  }
}

export default {
  set: __QUASAR_SSR_SERVER__ !== true && client.is.mobile === true && (
    client.is.nativeMobile === true
    || client.is.winphone === true || client.is.safari === true
    || client.is.webkit === true || client.is.vivaldi === true
  )
    ? hexColor => {
        const val = hexColor || getCssVar('primary')

        if (client.is.nativeMobile === true && window.StatusBar) {
          window.StatusBar.backgroundColorByHexString(val)
        }
        else {
          setColor(val)
        }
      }
    : noop,

  install ({ $q }) {
    $q.addressbarColor = this
    $q.config.addressbarColor && this.set($q.config.addressbarColor)
  }
}
```

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.0.0-beta.11.
